docs/guides/advanced/fetcher.md

```diff
@@ -303,7 +303,7 @@ try {
 
 ## Authentication
 
-The authentication you configure for your Pack is automatically applied to fetcher requests, with no extra code needed. For example, if you have setup `HeaderBearer` authentication, an `Authorization` header with the user's token will be automatically added to your fetcher requests. This is only done for formulas that use a connected account: those that have a `connectionRequirement` of `REQUIRED`, or `OPTIONAL` and the user opted to selected an account.
+The [authentication][authentication] you configure for your Pack is automatically applied to fetcher requests, with no extra code needed. For example, if you have setup `HeaderBearer` authentication, an `Authorization` header with the user's token will be automatically added to your fetcher requests. This is only done for formulas that use a connected account: those that have a `connectionRequirement` of `REQUIRED`, or `OPTIONAL` and the user opted to selected an account.
```

Comment:
Total nit, I generally think of "setup" as a noun and "set up" as a verb.

Reply:
Good catch, fixed.